### DISCUSSION TITLE
Fresh install, insert mod, cursor is overlay by the filename path prompts

### DISCUSSION THREAD

**Discussion by laowalter** [NvChad/NvChad#1659] [General]
New to Nvchad, fresh install, in insert mod , got this how to remove? 

https://www.imgpaste.net/image/KExWG2

thx

**Comment by siduck** [comment]
are you using kitty or alacritty by chance?

**Reply by laowalter** [reply]
> are you using kitty or alacritty by chance?

Thanks for reply, I use xterm. Gentoo Linux

**Reply by siduck** [reply]
does this happen with a non nvchad config?

**Reply by laowalter** [reply]
> does this happen with a non nvchad config?

No. just in nvchad. in ssh term or in turbovnc, btw I use putty and turbo vnc client.

**Reply by laowalter** [reply]
Want to disable this plugin temporarily, do not know which one makes it happen.

**Reply by siduck** [reply]
hmm does it occur when you use nvchad in your OS's nvim?

**Reply by laowalter** [reply]
Let me try.

**Reply by laowalter** [reply]
Just attach a real monitor and use nvcad directly in my OS nvim,  yes, it works well.

**Reply by siduck** [reply]
hmm do u face this issue with astrovim or lunarvim? :thinking:

**Reply by laowalter** [reply]
Any advice? what should I config for putty or turbovnc.  I always hit those special bugs. :(

**Reply by laowalter** [reply]
> hmm do u face this issue with astrovim or lunarvim? 🤔
did not try those, but maybe I can, then give you feed back.

**Reply by laowalter** [reply]
just tried, astrovim is good.

**Reply by siduck** [reply]
hmm, create an issue :)

**Reply by siduck** [reply]
> Any advice? what should I config for putty or turbovnc. I always hit those special bugs. :(

idk those things :grin:

**Reply by laowalter** [reply]
@siduck thanks for your help!

**Reply by siduck** [reply]
> @siduck thanks for your help!

dont forget to create the issue, i think ive seen this issue occurring many times, dont know how some nvchad users fixed it. Probably some bug lurking here :fearful: 

For some quick help in my absence, you can ask help for in our communities over discord, telegram, matrix

![image](https://user-images.githubusercontent.com/59060246/199394227-635e489a-316f-4948-a204-34431de2e410.png)

**Reply by laowalter** [reply]
An issure is opened. :) https://github.com/NvChad/NvChad/issues/1653
